user_mgr.cpp

```diff
@@ -258,8 +258,7 @@ void UserMgr::throwForMaxGrpUserCount(
     }
     else
     {
-        if (usersList.size() > 0 && (usersList.size() - getIpmiUsersCount()) >=
-                                        (maxSystemUsers - ipmiMaxUsers))
+        if (usersList.size() > 0 && (usersList.size() >= maxSystemUsers))
```

Comment:
Why is the check (usersList.size() > 0) needed, and can it be removed?
The old code had a funny/odd way to check the limit.  It is like they reserved the 15 IPMI users from to the total number of users allowed (maxSystemUsers).  I didn't see anything about that in https://github.com/openbmc/docs/blob/master/architecture/user-management.md and I will check with the community to see if this change should be upstreamed.